pysindy/feature_library/pde_library.py

```diff
@@ -107,6 +116,8 @@ def __init__(
         library_ensemble=False,
         ensemble_indices=[0],
         periodic=False,
+        implicit_terms=False,
+        multiindices=None,
```

Comment:
@znicolaou , what are the use cases for someone to pass `multiindices` directly?